specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/virtualWan.json

```diff
@@ -3197,6 +3435,17 @@
         "securityProviderName": {
           "type": "string",
           "description": "The Security Provider name."
+        },
+        "virtualHubRouteTableV2s": {
```

Comment:
Adding properties to existing responses is a breaking change and requires a new API version:
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo